boards/shields/esp_8266/doc/index.rst

```diff
@@ -65,7 +65,7 @@ ESP8266 AT Bin 1.7.x available requires the following partition configuration
 to works:
 
 .. image:: ./esp_at_173.png
-   :width: 720px
+   :width: 678px
```

Comment:
813?

Reply:
I selected lower number than 720 to decrease a little bit the images. I'm complete open here, may you have any suggestion?

Reply:
ok, I misunderstood your intention. That is fine